dnf/base.py

```diff
@@ -2285,19 +2287,71 @@ def provides(self, provides_spec):
                                for prefix in ['/bin/', '/sbin/', '/usr/bin/', '/usr/sbin/']]
         return self.sack.query().filterm(file__glob=binary_provides), binary_provides
 
-    def _merge_update_filters(self, q, pkg_spec=None, warning=True):
+    def add_security_filters(self, cmp_type, types=(), advisory=(), bugzilla=(), cves=(), severity=()):
+        #  :api
+        """
+        It modifies results of install, upgrade, and distrosync methods according to provided
+        filters.
+
+        :param cmp_type: only 'eq' or 'gte' allowed
+        :param types: List or tuple with strings. E.g. 'bugfix', 'enhancement', 'newpackage',
+        'security'
+        :param advisory: List or tuple with strings. E.g.Eg. FEDORA-2201-123
+        :param bugzilla: List or tuple with strings. Include packages that fix a Bugzilla ID,
+        Eg. 123123.
+        :param cves: List or tuple with strings. Include packages that fix a CVE
+        (Common Vulnerabilities and Exposures) ID. Eg. CVE-2201-0123
+        :param severity: List or tuple with strings. Includes packages that provide a fix
+        for an issue of the specified severity.
+        """
+        cmp_dict = {'eq': '__eqg', 'gte': '__eqg__gt'}
```

Comment:
Is there a reason why we need special `__eqg__gt` comparator? Doesn't current `__gte` cover it already?

Reply:
Good point - I want to provide a new behavior without changing of meaning of other comperators